app.py

```diff
@@ -8,46 +9,45 @@
 app = Flask(__name__)
 auth_api = "https://auth.verygoodsecurity.com/auth/realms/vgs/protocol/openid-connect/token"
 
-VAULT_ID = os.environ.get('VAULT_ID')
+VAULT_ID = os.environ.get('PAYMENT_ORCH_VAULT_ID')
 
 CORS(app)
 
-def get_access_token(scope):
+def get_access_token():
     data = {
-        'client_id': os.environ.get('MULTIPLEXING_CLIENT_ID'),
-        'client_secret': os.environ.get('MULTIPLEXING_CLIENT_SECRET'),
+        'client_id': os.environ.get('PAYMENT_ORCH_CLIENT_ID'),
+        'client_secret': os.environ.get('PAYMENT_ORCH_CLIENT_SECRET'),
         'grant_type': 'client_credentials', 
-        'scope': scope,
     }
     response = requests.post(auth_api, data=data)
-    print(response.json())
     return response.json()
         
 @app.route("/")
 def index():
-    access_token = get_access_token("financial-instruments:write")
-    return render_template('./index.html', tnt=VAULT_ID, accessToken=access_token.get('access_token'))
-@app.route("/access")
-def access():
-    access_token = get_access_token("financial-instruments:write transfers:write orders:write")
-    print(access_token)
-    return access_token.get('access_token')
+    return render_template('./index.html', customerVaultId = os.environ.get('CUSTOMER_VAULT_ID'))
 
-@app.route("/transfers", methods=['POST'])
-def transfer():
-    req = request.get_json()
-    access_token = get_access_token("transfers:write")
+@app.route("/checkout", methods=['POST'])
+def checkout():
+    # ==> Save aliassed credit card to DB here <==
+    access_token = get_access_token()
     headers = {
         "Content-Type": "application/json",
         "Authorization": "Bearer {}".format(access_token['access_token'])
     }
+    fin_instr_data = request.get_json()
+    fin_instr = requests.post('https://{}.sandbox.verygoodproxy.com/financial_instruments'.format(VAULT_ID),
+        headers = headers,
+        json = fin_instr_data
+    )
+    
     transfers_data = {
         "amount": 1 * 100,
         "currency": "USD",
-        "source": req['financial_instrument'],
+        "source": fin_instr.json()['data']['id'],
     }
-    tr = requests.post('https://{}.sandbox.verygoodproxy.com/transfers'.format(VAULT_ID),
-        headers=headers,
-        json=transfers_data
+    tansfer = requests.post('https://{}.sandbox.verygoodproxy.com/transfers'.format(VAULT_ID),
```

Comment:
I'd suggest to extract URL into variable and add a comment like "Your Payment Orchestration API URL here" since it can be different from customer to customer